docs/html/development/contributing.rst

```diff
@@ -40,7 +40,7 @@ Automated Testing
 =================
 
 All pull requests and merges to 'master' branch are tested using `Travis CI`_
-and `Appveyor CI`_ based on our `.travis.yml`_ and `.appveyor.yml`_ files.
+based on our `.travis.yml`_ file.
```

Comment:
while you're modifying this you might as well mention Azure DevOps & GitHub Actions and maybe add a link to our CI documentation ?

Reply:
Done in the latest commit